vpr/src/route/router_lookahead.cpp

```diff
@@ -87,10 +87,106 @@ float ClassicLookahead::classic_wire_lookahead_cost(int inode, int target_node,
     return (expected_cost);
 }
 
+float ClassicLookahead::get_expected_delay(int inode, int target_node, const t_conn_cost_params& params, float R_upstream) const {
```

Comment:
Shouldn't `get_expected_cost` be implemented in terms of `get_expected_delay` / `get_expected_cong` or `get_expected_delay_and_cong`? Should `get_expected_cost` even exist anymore?

Reply:
You're right, as well `get_expected_delay` / `get_expected_cong` could be implemented to also only use `get_expected_delay_and_cong` to eliminate code redundancy. I might just remove them altogether.